go/performance/benchmarks/helpers.go

```diff
@@ -0,0 +1,199 @@
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"strconv"
+	"strings"
+
+	"github.com/liquidata-inc/dolt/go/libraries/doltcore/sql"
+	"github.com/liquidata-inc/dolt/go/store/types"
+)
+
+// Container is used to correctly format sql strings
+type Container struct {
+	c []string
+}
+
+// NewContainer creates a new SQLContainer
+func NewContainer(format string) *Container {
+	c := make([]string, 3)
+	switch format {
+	case sqlExt:
+		c[0] = "("
+		c[2] = ")"
+	case jsonExt:
+		c[0] = "{"
+		c[2] = "}"
+	default:
+		log.Fatalf("cannot create new container, unsupported format %s \n", format)
+	}
+	return &Container{c: c}
+}
+
+// InsertPayload returns the SQLContainer with the payload inserted, separated by the separator
+func (sc *Container) InsertPayload(payload []string, separator string) string {
+	sc.c[1] = strings.Join(payload, separator)
+	return strings.Join(sc.c, "")
+}
+
+func getColValue(row []string, colIndex int, col *SeedColumn, format string) string {
+	switch col.GenType {
+	case increment:
+		return genNomsTypeValueIncrement(row, colIndex, col, format)
+	case random:
+		return getNomsTypeValueRandom(col, format)
+	default:
+		log.Fatalf("cannot get column value, unsupported gen type %s \n", col.GenType)
+	}
+	return ""
+}
+
+func genNomsTypeValueIncrement(row []string, colIndex int, col *SeedColumn, format string) string {
+	switch col.Type {
+	case types.IntKind:
+		if len(row) > 0 {
+			old, err := strconv.Atoi(row[colIndex])
+			if err != nil {
+				log.Fatalf(err.Error())
+			}
+			return fmt.Sprintf("%d", old+1)
+		}
+		return "1"
+	default:
+		log.Fatalf("cannot generate incremental value, unsupported noms type %s \n", col.Type.String())
+	}
+	return ""
+}
+
+func getNomsTypeValueRandom(col *SeedColumn, format string) string {
+	switch col.Type {
+	case types.IntKind:
+		return fmt.Sprintf("%d", rand.Intn(1000))
+	case types.StringKind:
+		return getRandomString(format)
+	default:
+		log.Fatalf("cannot generate random value, unsupported noms type %s \n", col.Type.String())
+	}
+	return ""
+}
+
+func getRandomString(format string) string {
+	lengths := []int{3, 5, 10, 15, 25, 35, 80, 100}
+	letters := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	b := make([]byte, lengths[rand.Int()%len(lengths)])
+	for i := range b {
+		b[i] = letters[rand.Int63()%int64(len(letters))]
+	}
+
+	switch format {
+	case sqlExt:
+		return fmt.Sprintf(`"%s"`, b)
+	default:
+		return string(b)
+	}
+}
+
+func getJSONRow(strs []string, cols []*SeedColumn, format string) string {
+	if len(strs) != len(cols) {
+		log.Fatalf("values should be the length of columns. values: %+v, columns: %+v \n", strs, cols)
+	}
+
+	payload := make([]string, 0)
+	for i, col := range cols {
+		load := fmt.Sprintf("\"%s\":\"%s\"", col.Name, strs[i])
+		payload = append(payload, load)
+	}
+
+	container := NewContainer(format)
+	return container.InsertPayload(payload, ",")
+}
+
+func getSQLRow(strs []string, cols []*SeedColumn, tableName, format string) string {
```

Comment:
Weird to me to thread `format` here on a (seemingly) format-specific method. Thoughts?

Reply:
Yea this is unnecessary, will change!